### DISCUSSION TITLE
TypeLoadException after adding Theme.Template

### DISCUSSION THREAD

**Discussion by Chatpard** [OrchardCMS/OrchardCore#16091] [Q&A]
Hi,
Using the Getting Started instructions. Build Cms.Web with target 3.1. Works fine.
Using CLI to add MyTheme.OrchardCore with target 3.1, referencing it in the main application brakes the application with error message 
TypeLoadException: Could not load type 'Microsoft.ExtensionsDependencyInjection.OrchardCoreBuilderExtensions' from assembly 'OrchardCore.Data, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'
The error occures in Startup.cs on line services.AddOrchardCms()
Any clue how to solve this error ?
Thanks

**Comment by deanmarcussen** [comment]
You have probably refererenced different versions of the cloudsmith packages, as the method it is refering to was moved, perhaps a month ago to a different project.

Check your versions are all at the same across all your .csproj files, do a dotnet clean and dotnet restore, and if it is still erroring post the version of OC you are using.
